explorer/explorer.go

```diff
@@ -513,6 +534,16 @@ func (exp *explorerUI) Store(blockData *blockdata.BlockData, msgBlock *wire.MsgB
 		go exp.updateDevFundBalance()
 	}
 
+	// Update the charts data after every five blocks or if no charts data
+	// exists yet. Do not update the charts data if blockchain sync is running.
```

Comment:
This comment might require some editing unless when the charts data is empty then `cacheChartsData.Height() == -1`.

Reply:
BTW, If the above is true then the first time 
`func (exp *explorerUI) Store(blockData *blockdata.BlockData, msgBlock *wire.MsgBlock) error {` 
runs 
`go exp.prePopulateChartsData()` 
will always run because the charts data is empty then.

Reply:
Also how does the height become -1 unless you are setting it somewhere?

Reply:
I have seen this function:
``` Golang
func (c *chartDataCounter) height() int64 {
	if c.Data == nil {
		return -1
	}
	return c.updateHeight
}
```
And in my opinion (not yet tested) the previous state (bug) might not yet be fixed.

Reply:
It actually doesn't run after my test. Let me look at the code again to see what I missed.

Reply:
I just received this a.m.  maybe that broke something, but I think it's ok

Reply:
But it is correct that the first time Store runs it will run prepopulate. We have it explicitly run initially in main, along with another request near the end of main that ends up not running a duplicate query (most of the time).  But the design should consider consumers of the explorer package who may not do things like we do in main.  Also, the query might fail or timeout, so it's prudent to verify it's done.